src/test/java/uk/gov/digital/ho/hocs/audit/SqsConfigurationTest.java

```diff
@@ -20,21 +18,21 @@ public void setup() {
 
     @Test
```

Comment:
If you remove those silly checks (remember, no git blame), we can remove these silly tests 😁

Reply:
Removed as no longer required.